Source/WebKit/GPUProcess/graphics/RemoteRenderingBackend.cpp

```diff
@@ -94,8 +94,9 @@ using namespace WebCore;
 bool isSmallLayerBacking(const ImageBufferParameters& parameters)
 {
     const unsigned maxSmallLayerBackingArea = 64u * 64u; // 4096 == 16kb backing store which equals 1 page on AS.
+    auto checkedArea = ImageBuffer::calculateBackendSize(parameters.logicalSize, parameters.resolutionScale).area<RecordOverflow>();
```

Comment:
Actually I am not very sure if this issue should be addressed in `calculateBackendSize` instead 🤔